EventPlanner/app/src/main/res/values/strings.xml

```diff
@@ -101,5 +101,7 @@
     <string name="number_of_results">Number of results: 10</string>
     <string name="log_out">Log out</string>
     <string name="incorrect_credentials">Incorrect email or password</string>
+    <!-- TODO: Remove or change this placeholder text -->
```

Comment:
Delete this, sometimes it causes unecessary errors because it generates itself automatically and created duplicates